scripts/pre.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+. /apps/lmod/lmod/init/sh
+module purge
+
+#module load contrib miniconda3
+module use -a /contrib/miniconda3/modulefiles
+module load miniconda3
+conda activate pygraf
+
+## conda activate adb_graphics
+
+module list
+
```

Comment:
The regional_workflow stores this type of script under the modulefiles subdirectories. This might fit better there as a `python_ensmean.local` file?

Reply:
Fixed. Thank you!